libethereum/State.cpp

```diff
@@ -336,6 +336,13 @@ void State::incNonce(Address const& _addr)
 		m_cache[_addr] = Account(requireAccountStartNonce() + 1, 0);
 }
 
+void State::setNonce(Address const& _addr, u256 const& _nonce)
```

Comment:
Exactly. This method was introduces just before this PR, but I forgot to actually provide the implementation. See https://github.com/ethereum/cpp-ethereum/pull/3444.